File: 0171-Title-To-Number/0171.py
class Solution:
  def titleToNumber(self, columnTitle: str) -> int:
    number, multiple = 0, 1
    for i in range(len(columnTitle) - 1, -1, -1):
      k = ord(columnTitle[i]) - ord("A") + 1
      number += k * multiple
      multiple *= 26
    return number

File: 101-200/0171-Title-To-Number/0171.java
class Solution {
  public int titleToNumber(String columnTitle) {
    int number = 0;
    int multiple = 1;
    for (int i = columnTitle.length() - 1; i >= 0; i--) {
      int k = columnTitle.charAt(i) - 'A' + 1;
      num += k * multiple;
      multiple *= 26;
    }
    return number;
  }
}